src/library_promise.js

```diff
@@ -221,6 +221,9 @@ mergeInto(LibraryManager.library, {
     var promises = idsToPromises(idBuf, size);
 #if RUNTIME_DEBUG
     dbg('emscripten_promise_any: ' + promises);
+#endif
+#if ASSERTIONS
+    assert(typeof Promise.any !== 'undefined', "Promise.any does not exist");
```

Comment:
We could do something more like this.. but its probably not worth it?: 

```
#if ENV_MAY_BE_NODE
if (ENV_IS_NODE && nodeIsOld) {
  abort("helpful message")
}
#endif
```

Reply:
Yeah, just having this assertion seems more general. For example it would work in old browsers that don't support Promise.any as well. I'd be happy to make the assertion message more useful, but I think the current message at least delivers the most important piece of information.